librz/core/cmd/cmd.c

```diff
@@ -1127,6 +1127,11 @@ RZ_API int rz_core_cmd_pipe_old(RzCore *core, char *rizin_cmd, char *shell_cmd)
 	return ret;
 }
 
+static void rz_pipe_stack_fini(void *e, void *user) {
+	(void)user;
+	free(e);
+}
```

Comment:
i do not think you need to do this since rz_vector does not allocate items as pointers.